compiler_opt/rl/data_reader_test.py

```diff
@@ -88,23 +87,29 @@ def setUp(self):
         maximum=20)
     super().setUp()
 
-  @parameterized.named_parameters(
-      ('SequenceExampleDatasetFn',
-       data_reader.create_sequence_example_dataset_fn),
-      ('TFRecordDatasetFn', data_reader.create_tfrecord_dataset_fn))
-  def test_create_dataset_fn(self, test_fn):
-    example = _define_sequence_example(
-        self._agent_name, is_action_discrete=True)
+  def _create_sequence_example_datasource(self, example):
+    return [example.SerializeToString() for _ in range(100)]
```

Comment:
nit: [example.SerializeToString()] * 100  since you don't need to rerun identical function 100 times here.